sql-plugin/src/main/330db/scala/com/nvidia/spark/rapids/330db/RapidsShuffleManager.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2023, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids.spark330db
```

Comment:
This file should be in com/nvidia/spark/rapids/spark330db rather than com/nvidia/spark/rapids/330db per this package declaration, and that would make it consistent with the other RapidsShuffleManager instances.